java/ql/src/Advisory/Documentation/ImpossibleJavadocThrows.ql

```diff
@@ -11,22 +11,18 @@
 
 import java
 
-RefType getTaggedType(ThrowsTag tag) {
+Class getTaggedType(ThrowsTag tag) {
   result.hasName(tag.getExceptionName()) and
-  exists(ImportType i | i.getFile() = tag.getFile() | i.getImportedType() = result)
+  result = tag.getFile().(CompilationUnit).getATypeInScope()
 }
 
-predicate canThrow(Callable callable, RefType exception) {
-  exists(string uncheckedException |
-    uncheckedException = "RuntimeException" or uncheckedException = "Error"
-  |
-    exception.getAnAncestor().hasQualifiedName("java.lang", uncheckedException)
-  )
+predicate canThrow(Callable callable, Class exception) {
```

Comment:
Same question re: throwing things by an interface type

Reply:
All exception types are classes though, so you cannot declare an interface type to be thrown, can you?
(neither in Kotlin, right?)

Reply:
Doh right, I had `Exception` in mind as an interface. Never mind.